src/arroz-chaufa.md

```diff
@@ -2,7 +2,7 @@
 
 Peruvian-chinese dish. Easy to cook just add and mix everything.
 
-![Arroz Chaufa](pix/arroz_chaufa-2.webp)
+![Arroz Chaufa](pix/arroz-chaufa-2.webp)
```

Comment:
```suggestion
![Arroz Chaufa](data/pix/arroz-chaufa-2.webp)
```
Fixed Error

Reply:
It is correct without the `data`. The webserver file structure is slightly different to the repository, so you don't need the `data` folder. Yes, it looks bad when you visit the markdown file here on github, but that's how it is

Reply:
> It is correct without the `data`. The webserver file structure is slightly different to the repository, so you don't need the `data` folder. Yes, it looks bad when you visit the markdown file here on github, but that's how it is

thanks!